marketdata/stream/subscription.go

```diff
@@ -154,6 +154,13 @@ func (cc *CryptoClient) SubscribeToOrderbooks(handler func(CryptoOrderbook), sym
 	return cc.client.handleSubChange(true, subscriptions{orderbooks: symbols})
 }
 
+func (cc *CryptoClient) SubscribeToFuturesPricing(handler func(pricing CryptoFuturesPricing), symbols ...string) error {
```

Comment:
Should be `SubscribeToPerpPricing`. Same for unsubscribe and (less importantly) for the unexported methods / handlers / etc.